include/eve/module/real/core/function/regular/simd/x86/convert_256.hpp

```diff
@@ -30,6 +30,14 @@ namespace eve::detail
     {
       return v0;
     }
+    else if constexpr( current_api >= avx2 && (N::value == 4)
+                       && std::is_integral_v<In> && (sizeof(In) == 8)
```

Comment:
Matter of style and compile-time, prefer using `std::integral` concept instead of the traits.